multihost_test/Sanity/test_pam_pwquality.py

```diff
@@ -0,0 +1,149 @@
+
+import pytest
+import subprocess
+import os
+
+
+def exceute_cmd(multihost, command):
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.tier2
+class TestPamBz(object):
+    def test_pam_pwquality1(self, multihost):
+        """
+        :title: Sanity tests for pam_pwquality.so part1
+        :id: e30c75be-eaf9-11eb-9781-845cf3eff344
+        """
+        exceute_cmd(multihost, 'yum install -y expect')
+        assert 'password' in exceute_cmd(multihost,
+                                         "grep system-auth "
+                                         "/etc/pam.d/passwd").stdout_text
+        exceute_cmd(multihost, "cp -vf /etc/pam.d/system-auth "
+                               "/etc/pam.d/system-auth_anuj")
+        exceute_cmd(multihost, "useradd local_anuj")
```

Comment:
can adding removing this user be a fixture?

Reply:
DOne